### PR TITLE
Task/responsive

### LINKED ISSUE #45: task/ responsive web design
## Description (Text, Scribble, Wireframe, Design)
Currently the app has only been optimized for mobile devices. It should be updated, so every device and screen size offers a good UX. the max-width may be limited.

## Acceptance criteria
- [x] the app offers a good UX on every screen size.
- [x] the app can be used with touch or mouse controls.

## Tasks
- [x] update styling for all necessary components
- [x] update tests
- [x] update Storybook (if necessary)

## Size
medium

### PR DESCRIPTION
**Pull request by karstenlenz**
Please review my code. Media queries have been added to improve usability and look on larger screens.

closes #45

### CODE REVIEW

File: src/Homepage/__snapshots__/Homepage.test.js.snap [hunk 1]
```diff
@@ -37,6 +37,7 @@ exports[`The Homepage component renders correctly 1`] = `
     >
       Was sind Persönlichkeitsstile?
       <img
+        alt="auf/zuklappen"
```
Comment (aruppert) [inline src/Homepage/__snapshots__/Homepage.test.js.snap:40]: Is this actionable? If so I would add "icon", "button" or similar so it's clear for the visually impaired with screen readers.  ;)

**Review by aruppert — APPROVED** [review]
lgtm :)

File: src/Homepage/Homepage.js [hunk 2]
```diff
@@ -137,13 +139,29 @@ const MirrorStyled = styled(MirrorImg)`
...
   margin: 0 auto;
-  width: 50%;
+  width: auto;
   height: 50%;
+  @media screen and (min-width: 750px) {
+    height: 125px;
+  }
+`
+const CardContainer = styled.section`
+  /* display: flex;
```
Comment (macfrei) [inline src/Homepage/Homepage.js:159]: If you don't use the style, delete it before merging.

File: src/common/Card.js [hunk 3]
```diff
@@ -6,10 +6,12 @@ export default function Card({ children }) {
...
 
 const CardBG = styled.div`
+  display: inline-block;
   background: white;
   border-radius: 12px;
   box-shadow: var(--primary-shadow);
   margin: 30px 0;
   padding: 15px;
   position: relative;
+  height: auto;
```
Comment (macfrei) [inline src/common/Card.js:16]: `height: auto;` means that the element height will depend on the children's height, which is already the case. So you don't need it here.

File: src/common/Card.js [hunk 4]
```diff
@@ -6,10 +6,12 @@ export default function Card({ children }) {
 }
 
 const CardBG = styled.div`
+  display: inline-block;
   background: white;
   border-radius: 12px;
   box-shadow: var(--primary-shadow);
   margin: 30px 0;
   padding: 15px;
   position: relative;
```
Comment (macfrei) [inline src/common/Card.js:15]: Why do you use `position: relative;` here?

**Review by macfrei — COMMENTED** [review]
I commented a bit on some styles you wrote. I like that you made your app responsive! 👍

File: src/common/Card.js (hunk 4 above)
Comment (karstenlenz) [reply src/common/Card.js:15]: I used it to position the buttons at the bottom, but it didn't work and I forgot to remove it here. Now I fixed the buttons so the `position:relative` can stay :)

**Pull request merged**